test/markup/diff/meta.expect.txt

```diff
@@ -0,0 +1,2 @@
+<span class="hljs-meta">@@ -1,8 +1,7 @@</span>
+<span class="hljs-meta">@@ -1,8 +1,7 @@</span> postfix
```

Comment:
The name of the test should really reflect what the content is or what it's testing, not the class used... so `line_range` might be a better name here, etc.  Same comment on the other meta-string test...

Reply:
I will merge both test into a single "git-patch" test. Will take some time but I will come up with a rebase...

Reply:
That would be much better! Thanks.